src/style.css

```
@@ -1,3 +1,68 @@
+:root {
+  --active: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  margin: 0;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo .logo-image {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: var(--active);
+}
+
+.nav__link.is-active {
+  display: flex;
+  position: relative;
+  color: var(--active);
+}
+
+.nav__link.is-active::after {
+  bottom: 0;
+  display: block;
+  position: absolute;
+  border-radius: 8px;
+  content: '';
+  height: 4px;
+  background-color: var(--active);
+  width: 100%;
+}
+
+.nav__list .nav__item:not(:first-child) .nav__link {
+  margin-left: 20;
```

```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
```
should work with shorter selector